app/controllers/event-invoice/review.js

```diff
@@ -0,0 +1,5 @@
+import Controller from '@ember/controller';
+
+export default class extends Controller {
+
+}
```

Comment:
I think we can remove this file.

Reply:
For now yes, I agree. When we link this to the payment component of paypal in the next PR, we'll use it.